### PR TITLE
feat(svelte-kit-scss): create top repos component 

### PR DESCRIPTION
**Pull request by asincole**
Closes: #674

### CODE REVIEW

File: svelte-kit-scss/src/routes/home/+page.svelte [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<script lang="ts">
```
Comment (flakolefluk) [inline svelte-kit-scss/src/routes/home/+page.svelte:1]: is this file intended to be here?

File: svelte-kit-scss/src/routes/(home)/+page.server.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        // TODO: how do I get the token
+        Authorization: `Bearer `
+      }
+    })
+    const data = (await response.json()) as UserReposApiResponse;
+    return {
+      topRepos: (mapUserReposToTopRepos(data))
+    }
+  } catch (err) {
+    console.error(err)
```
Comment (flakolefluk) [inline svelte-kit-scss/src/routes/(home)/+page.server.ts:23]: i think you should also either throw an error or return  an empty object, that should make `data` always defined in the page

File: svelte-kit-scss/src/lib/helpers/repositories.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import type {UserReposApiResponse} from "../interfaces";
+import type {UserReposState} from "../interfaces";
```
Comment (flakolefluk) [inline svelte-kit-scss/src/lib/helpers/repositories.ts:2]: can you make it a single import?

**Review by iansamz — APPROVED** [review]
LGTM
